matrix-sdk/src/main/java/org/matrix/androidsdk/rest/model/ReportContentParams.java

```diff
@@ -24,7 +24,7 @@
 public class ReportContentParams {
 
     // The event range from -100 “most offensive” to 0 “inoffensive”.
-    public List<Integer> score;
+    public Integer score;
```

Comment:
You can use the simple `int` type (but it's not blocking)

Reply:
oh yeah. sorry my Java is rusty!